spec/02-integration/05-proxy/33-cjson_encode_number_precision_spec.lua

```diff
@@ -0,0 +1,154 @@
+local helpers = require "spec.helpers"
+
+for _, strategy in helpers.each_strategy() do
+    describe("cjson encode number with defalut(14) precision [#" .. strategy .. "]", function()
```

Comment:
Should we test all the precisions with a loop?

Reply:
These two tests cover precision 14 and 16, but there is no test for 15. I think we can be more optimistic.